Sources/Sentry/SentryInAppLogic.m

```diff
@@ -48,7 +64,15 @@ - (BOOL)isClassInApp:(Class)targetClass
 
 + (BOOL)isImageNameInApp:(NSString *)imageName inAppInclude:(NSString *)inAppInclude
 {
-    return [imageName.lastPathComponent.lowercaseString hasPrefix:inAppInclude.lowercaseString];
+    return [SentryInAppLogic
+        isImageNameLastPathComponentInApp:imageName.lastPathComponent.lowercaseString
+                             inAppInclude:inAppInclude];
+}
+
++ (BOOL)isImageNameLastPathComponentInApp:(NSString *)imageNameLastPathComponent
+                             inAppInclude:(NSString *)inAppInclude
+{
+    return [imageNameLastPathComponent hasPrefix:inAppInclude];
```

Comment:
l: Having this function split seems like a bit of overkill.

Reply:
I fully agree, but the other one is public.